README.md

```diff
@@ -1,3 +1,5 @@
+[![codecov](https://codecov.io/gh/vitahlin/valkey/graph/badge.svg?token=AWQXLQXTXV)](https://codecov.io/gh/vitahlin/valkey)
```

Comment:
```suggestion
[![codecov](https://codecov.io/gh/valkey-io/valkey/graph/badge.svg)](https://codecov.io/gh/valkey-io/valkey)
```

Presumably? I know it's not loading, but I think it will once it starts working.

Reply:
This link requires the token param, needs to be copied from the codecov backend:
<img width="1222" alt="image" src="https://github.com/valkey-io/valkey/assets/6998610/15211a50-cf20-45b1-957c-596ba0d552d3">

The link to the graph of my forked project above is : https://app.codecov.io/gh/vitahlin/valkey/settings/badge

Valkey link maybe: https://app.codecov.io/gh/valkey-io/valkey/settings/badge

Reply:
Sorry for the delay, I was having issues getting codecov associated with the organization and I have now uploaded the CODE_COVERAGE_TOKEN

Reply:
I've re-updated valkey's badge address, thanks for the assistance!